docs/roadmap.md

```diff
@@ -0,0 +1,18 @@
+## Next Week
+
+- Allow table name to be customizable.
+- Incremental CDC-style backfill to avoid re-pulling full historical datasets.
+- Use JSONB for storing all of the data and allow user to create generated columns as necessary for future proofing. This ensures that read-only columns stay read-only because they are generated and also allow user to drop them without breaking the sync.
+- Automatically creating and dropping webhooks
+- Integration with ngrok as needed.
+
+## Next Month
```

Comment:
the engine library exports types as [vanilla TS objects](https://github.com/tx-stripe/stripe-sync-engine/blob/main/packages/sync-engine/src/schemas/types.ts) package. not sure how this will play with ORMs. if the ORM used does not support these native types or automated schema generation/introspection we still need a good story on how users will be able to join their own data with synced stripe data